packages/web-components/src/tooltip/tooltip.styles.ts

```diff
@@ -0,0 +1,147 @@
+import { css } from '@microsoft/fast-element';
+import { forcedColorsStylesheetBehavior } from '@microsoft/fast-foundation';
+import {
+  accentFillActiveBehavior,
+  accentFillHoverBehavior,
+  accentFillRestBehavior,
+  neutralFillHoverBehavior,
+  neutralFillInputActiveBehavior,
+  neutralFillInputHoverBehavior,
+  neutralFillInputRestBehavior,
+  neutralFillRestBehavior,
+  neutralFocusBehavior,
+  neutralForegroundRestBehavior,
+  neutralOutlineRestBehavior,
+} from '../styles/index';
+
+export const TooltipStyles = css`
+  :host {
+    position: relative;
+    contain: layout;
+    overflow: visible;
+    height: 0;
+    width: 0;
+  }
+
+  .tooltip {
+    box-sizing: border-box;
+    border-radius: calc(var(--corner-radius) * 1px);
+    border: calc(var(--outline-width) * 1px) solid transparent;
+    background: ${neutralForegroundRestBehavior.var};
+    color: ${neutralFillRestBehavior.var};
+    padding: 4px 12px;
+    height: fit-content;
+    width: fit-content;
+    font-family: var(--body-font);
+    font-size: var(--type-ramp-base-font-size);
+    line-height: var(--type-ramp-base-line-height);
+    white-space: nowrap;
+    // TODO: a mechanism to manage z-index across components
+    // https://github.com/microsoft/fast/issues/3813
+    z-index: 10000;
```

Comment:
I'm curious if this should be on the :host

Reply:
yes, this should be on :host